src/test/scala/encry/network/DeliveryManagerTests/DeliveryManagerRequestModifiesSpec.scala

```diff
@@ -0,0 +1,81 @@
+package encry.network.DeliveryManagerTests
+
+import java.net.InetSocketAddress
+import akka.actor.ActorSystem
+import encry.consensus.History.Older
+import encry.modifiers.InstanceFactory
+import encry.modifiers.history.{Block, Header, Payload}
+import encry.network.BasicMessagesRepo.{Handshake, ModifiersNetworkMessage}
+import encry.network.NetworkController.ReceivableMessages.DataFromPeer
+import encry.network.NodeViewSynchronizer.ReceivableMessages.{HandshakedPeer, OtherNodeSyncingStatus, RequestFromLocal}
+import encry.network.PeerConnectionHandler.{ConnectedPeer, Incoming}
+import encry.settings.EncryAppSettings
+import encry.utils.CoreTaggedTypes.ModifierId
+import encry.view.EncryNodeViewHolder.DownloadRequest
+import encry.view.history.EncryHistory
+import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpecLike}
+import encry.network.DeliveryManagerTests.DMUtils._
+
+class DeliveryManagerRequestModifiesSpec extends WordSpecLike with BeforeAndAfterAll
```

Comment:
Add test, which will simulate behavior when delivery manager sent more than "maxDeliveryChecks" and once again receive requestFromLocal